docs/smtchecker.rst

```diff
@@ -39,8 +39,15 @@ The potential warnings that the SMTChecker reports are:
 - ``<failing  property> happens here.``. This means that the SMTChecker proved that a certain property fails. A counterexample may be given, however in complex situations it may also not show a counterexample. This result may also be a false positive in certain cases, when the SMT encoding adds abstractions for Solidity code that is either hard or impossible to express.
 - ``<failing property> might happen here``. This means that the solver could not prove either case within the given timeout. Since the result is unknown, the SMTChecker reports the potential failure for soundness. This may be solved by increasing the query timeout, but the problem might also simply be too hard for the engine to solve.
 
-It is currently an experimental feature, therefore in order to use it you need
-to enable it via :ref:`a pragma directive<smt_checker>`.
+To enable the SMTChecker, you must select :ref:`which engine should run<smtchecker_engines>`,
+where the default is no engine.
+
+.. note::
+
+  Prior to Solidity 0.8.4, the default way to enable the SMTChecker was via
+  ``pragma experimental SMTChecker;``. That pragma has been deprecated, and
+  although it still enables the SMTChecker for backwards compatibility, it will
+  be removed in Solidity 0.9.0.
```

Comment:
Might be worth mentioning that selecting the engine enables it for all files while the old method allowed enabling it on a per-file basis.